### PR TITLE
Fix for: Error when adding code patch from side bar #373

### PR DESCRIPTION
**Pull request by 1wayposting**
This fixes this error:

https://github.com/yetone/avante.nvim/issues/373

I don't know if it causes other issues, but I haven't come across any while using this fix.

### CODE REVIEW

File: lua/avante/diff.lua [hunk 1]
```diff
@@ -516,7 +516,7 @@ function M.setup()
     group = augroup,
     pattern = "AvanteConflictDetected",
     callback = function(ev)
-      vim.diagnostic.enable(false, { bufnr = ev.buf })
```
Comment (aarnphm) [inline lua/avante/diff.lua:519]: what neovim version are you on?

This is the current API for 0.10 and 0.11 afaik

**Review by aarnphm — CHANGES_REQUESTED** [review]
see comments

**Comment by 1wayposting** [comment]
image

**Comment by aarnphm** [comment]
Screenshot 2024-08-29 at 14 11 34

I'm running 0.10.1 and it seems to work?

File: lua/avante/diff.lua (hunk 1 above)
Comment (aarnphm) [reply lua/avante/diff.lua:519]: https://github.com/neovim/neovim/blob/6ed530baa5050c4b63acd9d62c0884dd599c06fe/runtime/lua/vim/diagnostic.lua#L2010

I don't really know how to reproduce this error

Comment (aarnphm) [reply lua/avante/diff.lua:519]: Do you know what hashh you are building against?

Comment (aarnphm) [reply lua/avante/diff.lua:519]: from my end https://github.com/NixOS/nixpkgs/blob/05d8b74b04f1b16898c4b81da8eef65ac432e4e3/pkgs/by-name/ne/neovim-unwrapped/package.nix#L69

I suggest trying again with 0.10.1

**Comment by 1wayposting** [comment]
Ok yeah installed 0.10.1 and it works. Sorry about this, i saw this:

'avante.nvim is currently only compatible with Neovim 0.10.0 or later. Please ensure that your Neovim version meets these requirements before proceeding.'

and assumed i'd be alright

Thank you.

**Pull request closed without merging**